src/Nest/Document/Multiple/BulkAll/BulkAllResponse.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Newtonsoft.Json;
+
+namespace Nest
+{
+	/// <summary>
+	///
+	/// </summary>
+	public interface IBulkAllResponse
+    {
+        /// <summary>This is the Nth buffer.</summary>
+		long Page { get; }
+
+        /// <summary>The number of back off retries were needed to store this document.</summary>
+		int Retries { get; }
+    }
```

Comment:
What about `Errors` and `ItemsWithErrors`?  Can we accumulate them from the individual `Bulk()` calls?